app/models/post.rb

```diff
@@ -4,33 +4,66 @@ class Post < ActiveRecord::Base
   belongs_to :council
   has_and_belongs_to_many :profiles
   has_many :nominations
-  has_many :candidates 
+  has_many :candidates
   has_many :permission_posts
-  has_many :permissions, through: 'permission_posts'
+  has_many :permissions, through: :permission_posts
 
   # Scopes
-  scope :studierad, -> {where(elected_by: "Studierådet").order(council_id: :asc)}
-  scope :termins, -> {where(elected_by: "Terminsmötet").order(council_id: :asc)}
+  scope :studierad, -> { where(elected_by: "Studierådet").order(council_id: :asc) }
+  scope :termins, -> { where(elected_by: "Terminsmötet").order(council_id: :asc) }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.